kernel/src/snapshot.rs

```diff
@@ -230,18 +230,20 @@ impl Snapshot {
             .read_metadata(engine)?
             .ok_or(Error::MissingMetadata)?;
         let schema = metadata.schema()?;
-        let column_mapping_mode = match metadata.configuration.get(COLUMN_MAPPING_MODE_KEY) {
-            Some(mode) if protocol.min_reader_version >= 2 => mode.as_str().try_into(),
-            _ => Ok(ColumnMappingMode::None),
-        }?;
+        // FIXME
```

Comment:
What's the problem here, out of curiosity?

Reply:
oh yea just needed to delete now that i modified snapshot struct :)